.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
+  "parser": "babel-eslint",
+  "extends": "eslint:recommended",
```

Comment:
We're using `eslint-config-airbnb` in some other repos. Is that implementable with reasonable effort or does Strapi mess it up?